src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -1,11 +1,105 @@
 (ns status-im2.contexts.chat.messages.content.text.view
   (:require
+   [quo2.core :as quo]
+   [quo2.foundations.colors :as colors]
    [react-native.core :as rn]
-   [status-im.ui2.screens.chat.messages.message :as old-message]
-   [status-im2.contexts.chat.messages.link-preview.view :as link-preview]))
+   [status-im2.contexts.chat.messages.content.text.style :as style]
+   [status-im2.contexts.chat.messages.link-preview.view :as link-preview]
+   [utils.re-frame :as rf]))
+
+(defn mention-element
+  [from]
+  (rf/sub [:contacts/contact-name-by-identity from]))
+
+(defn render-inline
```

Comment:
it looks like a copy from old code? or do we really have all these in new designs ?

Reply:
We have them in the designs

Reply:
i believe there is no status tag anymore, also code doesn't look implemented as in designs , not sure about "emph"  "strong"  "strong-emph" and  "del" whats that, would be cool to have table in the description with the screenshot for each case

Reply:
@flexsurfer Removed the status tag. Here is the [table]( https://www.figma.com/file/wA8Epdki2OWa8Vr067PCNQ/Composer-for-Mobile?node-id=6891%3A392826&t=sfRoZgAVX0R4gHcD-0).

Emph is italic, strong is bold, del is strikethrough. These differences in names comes from status-go.

Not all cases are implemented yet. There is a second issue: https://github.com/status-im/status-mobile/issues/15004

Reply:
I believe it's no in the designs, but #community-channel should be supported, @John-44 @pedro-et is that correct?
@OmarBasem if that's the case, would you mind creating an issue tracking this please?

Reply:
Also, nested styling can have performance issues, so please make sure if implemented you test performance, we decided it not to implement it in the past because of these reasons, it might also be better to implement in a different way (in status-go), without nesting the parser

Reply:
@cammellos do you mean the status-tag? There is a second issue, I can update it to include that also: https://github.com/status-im/status-mobile/issues/15004

Reply:
@OmarBasem yes, I believe I had recently a discussion with @John-44 / @pedro-et and they didn't mention that we were to remove them, @ibrkhalil has also done some work on it recently (yet unmerged), so I think we will support them (I think desktop does, but I could be wrong), would you mind checking with them directly please?

Reply:
Sure, I will check them 👍